python-avd/pyavd/_cv/workflows/deploy_to_cv.py

```diff
@@ -50,7 +50,6 @@ async def deploy_to_cv(
 
     For any device referred under `configs`, `device_tags` and `interface_tags` the device:
     - The device must be present in the CloudVision Inventory and onboarded to the "Inventory & Topology Studio".
-        - TODO: See if we can relax the I&T requirement and add the device if it is missing.
```

Comment:
same this would make more sense in the other PR for cv deploy

Reply:
as mentioned in the PR text, I am cleaning up a few needless TODOs, since this was written during a complete TODO walkthrough.